proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -270,6 +270,9 @@ message WorkflowExecutionInfo {
     // (-- api-linter: core::0203::immutable=disabled
     //     aip.dev/not-precedent: field_behavior annotation is not yet used in this repo --)
     temporal.api.workflow.v1.WorkflowExecutionVersioningInfo versioning_info = 98;
+
+    // This flag indicates that this execution is a running because a previous run was reset.
+    bool is_reset_run = 99;
```

Comment:
You added originalExecutionRunID in https://github.com/temporalio/temporal/pull/6950. so is_reset_run == originalExecutionRunID != runID?

Reply:
Sorry. This PR is stale. Please don't review this one.

Reply:
To answer your question - yes. We don't need a special flag to determine if an execution is running after reset.
https://github.com/temporalio/temporal/blob/main/service/history/workflow/mutable_state_impl.go#L820